src/ui/FileViewer/CloseButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+
+export function CloseButton({ props }: {
+  props: FileViewerComponentProps;
+}) {
+  if (props.viewerType !== ViewerTypes.MULTI) {
+    const { onDelete, isByMe, disableDelete } = props;
+    return onDelete && isByMe && (
```

Comment:
`return onDelete && isByMe && ( <>markup</>)` might give you lint error:
You might wanna do => 
```
return onDelete && isByMe
	? <>markup ...</>
  : <></> 
```